### PR TITLE
Fixes the program never closing in Python 3.8.1 in Manjaro

### PR DESCRIPTION
**Pull request by apandada1**
1. For some reason, event.accept() did not work, so had to quit() instead
2. In Manjaro"QtWidQtWidgets.QMessageBox.Yes" is replaced by "PySide2.QtWidgets.QMessageBox.StandardButton.Yes". Now using str(reply) in if statement to make it work in both Manjaro and Ubuntu without getting an import error.

### CODE REVIEW

File: src/init.py [hunk 1]
```diff
@@ -30,7 +30,6 @@
 import util.write_config as WriteConfig
 import util.read_config as ReadConfig
 
-from PySide2.QtWidgets import QApplication
 
 
 class MyApplication(QtWidgets.QMainWindow):
```
Comment (LordAmit) [inline src/init.py]: won't this give an import error?

File: src/init.py [hunk 2]
```diff
@@ -41,10 +40,10 @@ def __assign_displays(self):
         self.no_of_displays = len(self.displays)
         self.no_of_connected_dev = self.no_of_displays
 
-        if self.no_of_displays is 1:
+        if self.no_of_displays == 1:
```
Comment (LordAmit) [inline src/init.py]: hm, why this change? python3 supports this; and it is more readable.

Comment (apandada1) [reply src/init.py]: It gave some warning in Python 3.8 that it might be removed in future

File: src/init.py (hunk 1 above)
Comment (apandada1) [reply src/init.py]: I don't think so (application already runs without error) because we are already importing whole QtWidgets in a previous line

File: src/init.py (hunk 2 above)
Comment (LordAmit) [reply src/init.py]: can you please put the warning here so that I can dig into it?

**Comment by apandada1** [comment]
./init.py:43: SyntaxWarning: "is" with a literal. Did you mean "=="?
  if self.no_of_displays is 1:

File: src/init.py (hunk 2 above)
Comment (apandada1) [reply src/init.py]: `./init.py:43: SyntaxWarning: "is" with a literal. Did you mean "=="?
  if self.no_of_displays is 1:`

Comment (apandada1) [reply src/init.py]: @LordAmit I misunderstood the warning. It's probably not going to be completely removed from python, but now is and == have different uses. Check this. https://adamj.eu/tech/2020/01/21/why-does-python-3-8-syntaxwarning-for-is-literal/

Comment (LordAmit) [reply src/init.py]: Yup; I understand. we better use ==.

File: src/init.py (hunk 1 above)
Comment (LordAmit) [reply src/init.py]: Which line? I do not see it. 🏳️

Comment (apandada1) [reply src/init.py]: This line. https://github.com/LordAmit/Brightness/blob/9ab45ba18ba6359b90c7d9009a9a76bf41bebce4/src/init.py#L22

Comment (LordAmit) [reply src/init.py]: Damnit github.. I thought it was line 30 was the first line. 🤦‍♂
